package.json

```diff
@@ -11,7 +11,7 @@
   "dependencies": {
     "@sentry/node": "~7",
     "archiver": "~5",
-    "bcrypt": "~5",
+    "bcrypt": "~5.1",
```

Comment:
See my comment on the issue: https://github.com/getodk/central-backend/issues/1006#issuecomment-1850884855. I don't think we really need to pin the minor version in package.json. It's enough that package-lock.json specifies the exact version to install. If we specify a minor version here, it'll be slightly harder to update to a later version in the future. Not a big deal though, we can always change it in package.json later if you prefer to merge this.